tests/test_check_functions.py

```diff
@@ -0,0 +1,66 @@
+import json
```

Comment:
This file also should have the copyright notice at top. You can copy it from any other `.py` file.